OWNERS

```diff
@@ -37,4 +37,11 @@ filters:
     required_reviewers:
       - kubernetes/dep-approvers
     labels:
-      - area/dependency
\ No newline at end of file
+      - area/dependency
+  # metrics.go files are sig-instrumentation related, and should be tagged
+  # and reviewed by sig-instrumentation
+  "metrics\\.go$":
+    required_reviewers:
+      - sig-instrumentation-reviewers
```

Comment:
there was some debate about this for dep approvers... 

for some reason we use a github team instead of an alias.
I don't remember why we'd do this if an alias worked, because the team means *everyone* is assigned (and can't unassign, team assignment mechanisms are broken).

I feel like if using an alias here worked properly, we'd be using it elsewhere.

cc @cblecker @dims

Reply:
I removed the reviewers portion, we’d be thrilled just to have auto-labeling for SIG related PRs.